src/main/scala/org/broadinstitute/dsde/agora/server/business/AgoraBusiness.scala

```diff
@@ -128,6 +124,18 @@ class AgoraBusiness {
     }
   }
 
+  private def validateMethodRef(method: AgoraEntity, username: String) = {
+    if (!AgoraEntityPermissionsClient.getEntityPermission(method, username).canRead &&
+      !AgoraEntityPermissionsClient.getEntityPermission(method, "public").canRead) {
+      throw new AgoraEntityNotFoundException(method)
```

Comment:
Do we really want to throw an entity not found exception? The config exists, but it's referenced method doesn't. So is that a ValidationException?  How is this exception being handled?